manifests/4.5/cluster-logging.v4.5.0.clusterserviceversion.yaml

```diff
@@ -418,27 +418,27 @@ spec:
       statusDescriptors:
       - description: The status for each of the Kibana pods for the Visualization component
         displayName: Kibana Status
-        path: visualization.kibanaStatus.pods
+        path: visualization.kibanaStatus[0].pods
```

Comment:
What happens when there is more then one status?  What happens when there is no status?

Reply:
@jcantrill unfortunately we may be past this update from the API perspective.. but I don't think the CLO will ever need to worry about more than one Kibana. This is leftover from the `ops` pattern. so it should only be `[0]`...

Reply:
Yes, I too think I should be only [0]

Reply:
@ewolinetz which is fine but what happens when there are no kibana status or any of the others for that matter.  I'm trying to ensure we don't introduce some error condition

Reply:
@jcantrill seems to be no worse than what's already seen https://github.com/openshift/cluster-logging-operator/issues/439